Resources/Prototypes/_Impstation/Catalog/Shipyard/civilian.yml

```diff
@@ -0,0 +1,32 @@
+- type: vessel
+  id: Botany
+  name: Botany Ship
+  description: A small botanic vessel.
+  price: 15000
+  path:
+  - /Maps/_Impstation/Shuttles/botany.yml
+  categories:
+  - Small
+  - Service
+
+- type: vessel
+  id: Medical
+  name: Medical Ship
+  description: A medium medical vessel.
+  price: 50000
+  path:
+  - /Maps/_Impstation/Shuttles/medical.yml
+  categories:
+  - Medium
+  - Medical
+
+- type: vessel
+  id: Reach
+  name: SS Reach II
```

Comment:
the actual ship name is NTSC Reach Transport <letters>-<numbers>. would be better to name this version, like, "NTSC Reach Transport QT-900" or something. or find a funny serial code for the end

Reply:
I just took the ship's name from the files but sure this version sound cooler

Reply:
yeah the way the ship naming works is a lot more complex than it really should be. but if you localhost start a round on reach youll see that the ship name is NTSC Reach Transport even tho the file says '{0} Reach Transport {1}'